app/objects/beatmap.py

```diff
@@ -665,7 +665,11 @@ async def _update_if_available(self) -> None:
                         bmap._parse_from_osuapi_resp(new_map)
                         updated_maps.append(bmap)
                     else:
-                        # map is the same, make no changes
+                        # map md5 is the same, but status maybe changed (qualified to ranked)
+                        if not old_map.frozen:  # frozen map should not update status
+                            old_map.status = RankedStatus.from_osuapi(
+                                new_map["approved"],
+                            )
```

Comment:
there's a `updated_maps.append(old_map)` right after these lines, and 
https://github.com/TrueRou/bancho.py/blob/f5590ff474177cc0d89382211ffc03f288fc5b4e/app/objects/beatmap.py#L719
self._save_to_sql() at the end.
Could you please check again if changes are not reflected to database?

Reply:
Trou is busy preparing for gaokao test and might not be available for quite a while.